src/index.html

```diff
@@ -16,17 +16,38 @@
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="form form__1">
+      <div class="input__wrapper">
+        <img
+          src="images/Search.svg"
+          alt="search-logo"
+          class="input__icon input__icon__1"
+        />
+        <input
+          class="input input__first"
+          type="text"
+          data-qa="big keypress"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
Use semantic tags where possible. Instead of using `div` for wrapping input elements, consider using `label` which is more appropriate for form input elements.